invenio_indexer/cli.py

```diff
@@ -52,3 +53,19 @@ def run(delayed, concurrency):
     else:
         click.secho("Indexing records...", fg='green')
         RecordIndexer().process_bulk_queue()
+
+
+@index.command()
```

Comment:
you can try to make `reindex` and `run` chainable (http://click.pocoo.org/5/commands/#multi-command-chaining).